app-emulation/fuse/Manifest

```diff
@@ -1,2 +1,2 @@
-DIST fuse-1.4.0.tar.gz 1652821 BLAKE2B b7d8c5cd8c5fb13e077029eeee6283c9d2a114057c83e29bacc1d99933b338011ce255b5df34b6039bcf03410af46bf586cb6d6ec038d53b8e83bad2308d2ed1 SHA512 e5d0e65820f54d0ac0447d50aa89ea4ba3e6bb717e270ae813c640a9b36fe015c68f194edacb5cbe8899ee7893c30cab4493ed297919e962e722e9d3e1f72001
```

Comment:
Please split the version bump into separate commits adding new version and removing old.